### ISSUE TITLE
Doesn't work with Video.js 5.0

### ISSUE THREAD

**Issue by walsh9** [walsh9/videojs-transcript#5]
It looks like they changed something about the way tracks load, so I need to update the plugin.

Quick workaround for anyone trying to get this to work right now.

1. Pass all the plugin initialization code as a function to videojs's ready callback. If you're working from the example, change the script on the `example.html` to something like this:

```javascript
var video = videojs('video');
video.ready(function () {
  var transcript = video.transcript();
  var transcriptContainer = document.querySelector('#transcript');
  transcriptContainer.appendChild(transcript.el());
});
```

2. You might also need to change the line of code in the plugin that says:

```javascript
my.settings = videojs.util.mergeOptions(defaults, options);
```

To:

```javascript
my.settings = videojs.mergeOptions(defaults, options);
```

**Comment by spaceoi** [comment]
There is another error reported by Chrome: Uncaught TypeError: Cannot read property 'activeCues' of undefined;
guess the track doesn't work?

**Comment by magoodaboy** [comment]
Even with those changes it seems to have an issue with activeCues because it can't find the tracks. Anything else that has to be changed to get this workings?

Thanks

**Comment by walsh9** [comment]
fixed by #9
